web_App/urls.py

```diff
@@ -15,11 +15,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-from hello import views
+from django.urls import path, include
+from hello import views 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path("", views.home, name="home"),
-
+    path('regi/', include('hello.urls')),
```

Comment:
`regi/` is not appropriate name. Make it a meaningful name cause this url path shows that what we are trying to achieve with this.